dot/state/service_integration_test.go

```diff
@@ -177,7 +179,7 @@ func TestService_StorageTriePruning(t *testing.T) {
 		Path:     t.TempDir(),
 		LogLevel: log.Info,
 		PrunerCfg: pruner.Config{
-			Mode:           pruner.Full,
+			// Mode:           pruner.Full,
```

Comment:
is this related to https://github.com/ChainSafe/gossamer/pull/2831?

Reply:
Yes since `pruner.Full` got removed. This test is to be unskipped and this line uncommented (modified) in #2381). I added a comment on the other PR to make sure I don't forget 😉